src/System.Net.Http/src/System/Net/Http/HttpRequestException.cs

```diff
@@ -6,7 +6,6 @@
 
 namespace System.Net.Http
 {
-    [Serializable]
     [SuppressMessage("Microsoft.Serialization", "CA2229")]
```

Comment:
?I believe that suppression is still valid because it's a warning that the type implements ISerializable

But if we're removing the Serializable attribute, then the type shouldn't be implementing ISerializable anymore.  So, why would there be a warning at all?